### PR TITLE
Feature - Move Stripe addon into mono repo

### LINKED ISSUE #1386: Move Stripe addon into monorepo

### PR DESCRIPTION
**Pull request by alecritson**

### CODE REVIEW

File: tests/stripe/ExampleTest.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+test('example', function () {
```
Comment (glennjacobs) [inline tests/stripe/ExampleTest.php]: I don't think we need this

File: tests/stripe/Feature/.gitkeep
Comment (glennjacobs) [inline tests/stripe/Feature/.gitkeep:1]: Remove?

File: .github/workflows/stripe_tests.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+name: Lunar Table Rate Shipping Tests
```
Comment (lguichard) [inline .github/workflows/stripe_tests.yml]: @alecritson Small typo error here ;-)

File: packages/stripe/.gitignore [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+public/storage
+public/hot
+
+# Laravel 5 & Lumen specific with changed public path
+public_html/storage
+public_html/hot
+
+storage/*.key
+.env
+Homestead.yaml
```
Comment (glennjacobs) [inline packages/stripe/.gitignore]: What's all these entries in the gitignore? They don't seem relevant.

**Review by glennjacobs — DISMISSED** [review]

**Review by glennjacobs — APPROVED** [review]

File: packages/stripe/composer.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  "authors": [
+    {
+      "name": "Lunar",
+      "homepage": "https://lunarphp.io/"
+    }
+  ],
+  "require": {
+      "php": "^8.1",
+      "lunarphp/core": "self.version",
+      "stripe/stripe-php": "^7.114",
```
Comment (adevade) [inline packages/stripe/composer.json]: Why the old version? Or that's not in the scope for this PR, I assume?

**Pull request merged**